frontend/src/component/incomingWebhooks/IncomingWebhooksEvents/IncomingWebhooksEventsModal.tsx

```diff
@@ -0,0 +1,180 @@
+import { Button, Link, styled } from '@mui/material';
+import { SidebarModal } from 'component/common/SidebarModal/SidebarModal';
+import { IIncomingWebhook } from 'interfaces/incomingWebhook';
+import { useIncomingWebhookEvents } from 'hooks/api/getters/useIncomingWebhookEvents/useIncomingWebhookEvents';
+import { Suspense, lazy } from 'react';
+import FormTemplate from 'component/common/FormTemplate/FormTemplate';
+import useUiConfig from 'hooks/api/getters/useUiConfig/useUiConfig';
+import { SidePanelList } from 'component/common/SidePanelList/SidePanelList';
+import { formatDateYMDHMS } from 'utils/formatDate';
+import { useLocationSettings } from 'hooks/useLocationSettings';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+
+const LazyReactJSONEditor = lazy(
+    () => import('component/common/ReactJSONEditor/ReactJSONEditor'),
+);
```

Comment:
Could the component be exported this way? so it's always lazily imported? Or is that an implementation detail more than what we should do every time?

Reply:
If I understand correctly you're asking whether we can export the lazy import, so that it is always lazy imported. I think it should always be up to the parent to decide how to import the component, but we can discuss this in the team.

Reply:
Yes exactly. I saw it being lazily imported twice in the same PR, so figured it was worth asking if this should be the standard way of doing this